README.md

```diff
@@ -48,6 +48,7 @@ This example creates an AssetReference that is restricted to having a specific C
 * Scripts/ColorChanger -  ColorChanger & ComponentReferenceColorChanger
   * The component type we chose to care about.
   * Note that this file includes a concrete version of the ComponentReference.  This is needed because if your game code just specified a ComponentReference<ColorChanger> it could not serialize or show up in the inspector.  This ComponentReferenceColorChanger is what makes serialization and the inspector UI work.
+  * Releasing a ComponentReference<TComponent> requires special work if not done through `ReleaseInstance()` in the ComponentReference<TComponent> class.
```

Comment:
I'd update this to something like:
Releasing a ComponentReference<TComponent> should be done through `ReleaseInstance()` in the ComponentReference<TComponent> class.  To release an instance directly, see our implementation of ReleaseInstance to understand the requirements.

Reply:
updated!